.vitest/vitest.shared.ts

```diff
@@ -3,6 +3,11 @@ import { configDefaults, defineConfig } from "vitest/config";
 
 export const sharedConfig = defineConfig({
   test: {
+    typecheck: {
+      enabled: process.env.TYPECHECK === "true",
+      only: process.env.TYPECHECK === "true",
+      ignoreSourceErrors: true,
```

Comment:
When testing, files that just don't typecheck, ie the multisig account
Will fail this step without this set as true
<img width="483" alt="image" src="https://github.com/user-attachments/assets/81ec7bc2-3a40-4f2e-9f86-0fbe5fcd07a8" />

Reply:
Ah we should just exclude this file from the test sets since later we'll delete this file. That way the other tests can at least get caught as well. We've had cases where code built no prob but it ruined some branch of the types and we didn't notice cuz the tests don't get type checked

Reply:
They already are excluded in the tests btw, just the typecheck will check non test files that are excluded.